README.md

```diff
@@ -26,6 +26,9 @@ conda install -c conda-forge numpy scipy matplotlib netCDF4 xarray dask bottlene
 ```
 
 ## Running the analysis
-  1. create a configuration file by copying and modifying `config.template` or one of the example files in `configs`
-  2. run: `./run_analysis.py config.myrun`, where `config.myrun` is the config file you created
-  3. If you want to run a subset of the analysis, you can either modify the `generate` option under `[output]` in the config file or use the `--generate` flag on the command line.  See `config.template` for more details.
+  1. create a configuration file (say `config.myrun`) by copying and modifying 
+     `config.default` or one of the example files in `configs`.  Note, you only
+     need include entries that are modified from `config.default` in your 
+     confguration file; entries from `config.default` will be used by default.
```

Comment:
I believe the typo is gone (but probably replaced with others).

You always have to at least set `baseDirectory` in `[output]`, so you always need a config file.  Also, the command line arguments currently give you an error if you don't supply a config file.